params/config_classic.go

```diff
@@ -60,6 +60,8 @@ var (
 		EIP145FBlock:  big.NewInt(9573000),
 		EIP1014FBlock: big.NewInt(9573000),
 		EIP1052FBlock: big.NewInt(9573000),
+		// EIP1283FBlock:   big.NewInt(9573000),
+		// PetersburgBlock: big.NewInt(9573000),
```

Comment:
I've just added these as documentation to reflect the related Constantinople config; there were issues with EIP1234 that came up in other places and this was part of my investigation.